pom.xml

```diff
@@ -10,26 +10,19 @@
     <packaging>jar</packaging>
 
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <camel.version>3.0.1</camel.version>
+        <camel.version>3.3.0</camel.version>
```

Comment:
I'm not familiar with how Camel's dependency model generally works, do you know what implications this will have for using the component with older versions of camel? Will we need to maintain separate builds if we intend to?

Reply:
Actually I think we should add <scope>provided</scope> to the camel-core dependency.

Reply:
That makes sense to me, @jam01 what do you think?

Reply:
Yeah I think that makes sense.

Reply:
Okay I think once you've changed the camel dependency to provided and verified that works we're all good, and since we're currently in SNAPSHOT no need to tick version, we'll do that when we release